troubadix/standalone_plugins/changed_packages/changed_packages.py

```diff
@@ -168,12 +165,16 @@ def main():
     for file in args.files:
         try:
             old_content = git("show", f"{args.start_commit}:{file}")
+            content = git("show", f"HEAD:{file}")
+            missing_packages, new_packages = compare(old_content, content)
         except CalledProcessError:
-            terminal.error(f"Could not find {file} at {args.start_commit}. ")
-            sys.exit(1)
-
-        content = git("show", f"HEAD:{file}")
-        missing_packages, new_packages = compare(old_content, content)
+            terminal.error(
+                f"Could not find {file} at {args.start_commit} or HEAD"
+            )
```

Comment:
```suggestion
            terminal.warn(
                f"Could not find {file} at {args.start_commit} or HEAD"
            )
```

Is is `warn` or `warning` ?

Reply:
`ConsoleTerminal` only has `warning´. Were you thinking of https://docs.python.org/3/library/logging.html#logging.warning, which also has `warn` (although being deprecated)?

Reply:
Updated code suggestion

Reply:
I'd argue that a file not existing at a previous state is not an error condition (and at best a warning), but I am not deep enough into this, so its up to you.

Reply:
The issue with not finding the previous state is that we cannot perform a comparison then and changes to the packages can slip through unnoticed.
I expected that not finding it only occurs, because a user chose the wrong start commit (e.g. where the NASL files have a different name).